pages/access/service/[serviceId].js

```diff
@@ -406,7 +464,12 @@ function MarketplaceService() {
               <AzureLogo />
             ) : params.row.cloud_provider === "gcp" ? (
               <GcpLogo />
-            ) : null
+            ) : (
+              <GridCellExpand
```

Comment:
Is it possible to reach this condition? Can cloud provider can have a value other than aws/gcp/azure?